.syncpackrc

```diff
@@ -0,0 +1,24 @@
+{
+  "dev": true,
+  "filter": ".",
+  "indent": "  ",
+  "overrides": true,
+  "peer": true,
+  "prod": true,
+  "resolutions": true,
+  "workspace": true,
+  "semverGroups": [],
+  "semverRange": "",
+  "sortAz": [
+    "contributors",
+    "dependencies",
+    "devDependencies",
+    "keywords",
+    "peerDependencies",
+    "resolutions",
+    "scripts"
+  ],
+  "sortFirst": ["name", "description", "version", "author"],
+  "source": [],
+  "versionGroups": []
+}
```

Comment:
Thanks for trying syncpack, [the shape of the config changed in 9.0.0](https://github.com/JamieMason/syncpack/releases/tag/9.0.0) but this should work for you:

```diff
{
-  "dev": true,
    "filter": ".",
    "indent": "  ",
-  "overrides": true,
-  "peer": true,
-  "prod": true,
-  "resolutions": true,
-  "workspace": true,
+  "dependencyTypes": [
+    "dev",
+    "overrides",
+    "peer",
+    "prod",
+    "resolutions",
+    "workspace"
+  ],
  "semverGroups": [],
  "semverRange": "",
  "sortAz": [
    "contributors",
    "dependencies",
    "devDependencies",
    "keywords",
    "peerDependencies",
    "resolutions",
    "scripts"
  ],
  "sortFirst": ["name", "description", "version", "author"],
  "source": [],
  "versionGroups": []
}
```

Reply:
Thanks a lot!